### PR TITLE
Object Property Access

### PR DESCRIPTION
**Pull request by cdifino**
- [x ] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

**Comment by cdifino** [comment]
Hi @corevo, @tourdedave! 

This is another proposal on how to implement object property access. I had some issues implementing verify/assert property access with the previous implementation of the feature. With this implementation, all the three scenarios work. Looking forward to your input. 

Regards, 

Camila

**Comment by corevo** [comment]
Thanks for helping out with this, do you mind also forward porting this to `master`?

We can't afford to regress this once the electron version will get off the ground.

**Comment by cdifino** [comment]
Yes @corevo ! I can port this to master :)

**Comment by cdifino** [comment]
I like the change, thank you! Did you actually made the change ? I can do it if not!

**Comment by corevo** [comment]
No, I didn't make any changes.

**Comment by corevo** [comment]
Don't worry about `master`, I was thinking about it, and I'm gonna do something else entirely.

Just fix the test and I'll merge it 🎉

**Comment by corevo** [comment]
I also completely forgot about the runner, I think it should work out of the box, we just need a test for it.

**Comment by cdifino** [comment]
we can add tests to the runner but the runner's property access already worked (I think using javascript)

**Comment by corevo** [comment]
I think if you just add another `side` file to the tests to check it out it'll be fine. (https://github.com/SeleniumHQ/selenium-ide/tree/v3/tests/examples)

File: package.json [hunk 1]
```diff
@@ -40,25 +40,26 @@
...
+    "jest": "^24.9.0",
+    "lerna": "^3.16.4",
+    "prettier": "^1.18.2",
+    "putasset": "^4.3.3",
+    "stylelint": "^9.10.1",
     "stylelint-config-prettier": "^4.0.0",
-    "stylelint-config-standard": "^18.2.0",
-    "stylelint-prettier": "^1.0.3",
-    "putasset": "^4.3.2",
-    "grizzly": "^4.0.1"
```
Comment (corevo) [inline package.json]: This seems to have been removed by accident

File: packages/selenium-side-runner/run.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+npm uninstall -g selenium-side-runner
```
Comment (corevo) [inline packages/selenium-side-runner/run.sh]: You don't need this if you run `npm i -g .` and you run `yarn watch` your globally installed `selenium-side-runner` will automatically update.

File: package.json [hunk 3]
```diff
@@ -40,25 +40,27 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.3.4",
-    "babel-eslint": "^10.0.1",
-    "babel-jest": "^24.3.1",
+    "@babel/core": "^7.6.0",
+    "@babel/preset-env": "^7.6.0",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (corevo) [inline package.json]: There's no need for both `@babel/core` and `babel-core` we should remove `babel-core`

File: packages/selenium-side-runner/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+run.sh
```
Comment (corevo) [inline packages/selenium-side-runner/.gitignore]: Not necessary anymore

File: packages/selianize/package.json [hunk 5]
```diff
@@ -26,6 +26,7 @@
   },
   "devDependencies": {
     "@babel/preset-env": "^7.3.4",
+    "ndb": "^1.1.4",
```
Comment (corevo) [inline packages/selianize/package.json]: Install this globally and not as a dependency (since it is not required anywhere in the library)

**Pull request closed without merging**